packages/cli/src/lib/config.ts

```diff
@@ -96,11 +96,20 @@ export async function loadCliConfig(options: Options) {
     });
     const frontendConfig = ConfigReader.fromConfigs(frontendAppConfigs);
 
+    const backendAppConfigs = schema.process(appConfigs, {
+      visibility: ['frontend', 'backend', 'secret'],
+      withFilteredKeys: options.withFilteredKeys,
```

Comment:
We can skip this option for these, we're not expecting anything to be filtered out anyway